lib/puppetserver/ca/action/import.rb

```diff
@@ -73,7 +73,8 @@ def run(input)
 
         def import(loader, settings, signing_digest)
           ca = Puppetserver::Ca::LocalCertificateAuthority.new(signing_digest, settings)
-          master_key, master_cert = ca.create_master_cert(loader.key, loader.certs.first)
+          ca.load_ssl_components(loader)
```

Comment:
would it be weird to pass the key and cert into `.new()`?

Reply:
Initially I wanted the caller to be able to specify more about how the `LocalCertificateAuthority` was created via named arguments, but I decided against adding more complexity to the constructor's arguments and opted for a way to set the key and cert via `load_ssl_components`. I'd consider that method to be the public accessor/setter for the class instance vars introduced in this PR.